tools/lint-docstring

```diff
@@ -0,0 +1,246 @@
+    after the text
+    """
+    folder_width, file_width, desc_width = columnwise_widths
+    folder_text, file_text, desc_text = column_text
+    folder_remaining_spaces = (folder_width - len(folder_text)) * SPACEBAR
+    file_remaining_spaces = (file_width - len(file_text)) * SPACEBAR
+    desc_remaining_spaces = (desc_width - len(desc_text)) * SPACEBAR
+    folder_name = folder_text + folder_remaining_spaces
+    file_name = file_text + file_remaining_spaces
+    file_desc = desc_text + desc_remaining_spaces
+    return (folder_name, file_name, file_desc)
+
+
+def write_to_file(list_of_new_files_to_add, current_doc_data, write_location) -> None:
+    """
+    Writes the list of rows containing folder name, file name(s) and their respective
+    descriptions to the developer-file-overview.md
+    """
+    current_doc_data[(write_location) : (write_location - 1)] = list_of_new_files_to_add
+    write_location = write_location + len(list_of_new_files_to_add) + 1
+    # write_location += 1
+    updated_data = "".join(current_doc_data)
+    with open(DEVELOPER_DOC, "w") as file:
+        file.write(updated_data)
+
+
+def extract_folder_file_structure() -> Dict[str, str]:
+    """
+    Returns dictionary containing folders and respective python files within them
+    """
+    # contains path to the folders to be parsed to search for files
+    folder_paths = {}
+    for path_to_folder in glob.glob(f"{ZULIPTERMINAL}/**", recursive=True):
+        directory_path = dirname(path_to_folder)
+        relative_path = relpath(directory_path, ROOT_DIRECTORY)
+        if directory_path != str(THEMES_PATH) and directory_path != str(SCRIPTS_PATH):
+            folder_paths[relative_path] = directory_path
+
+    folders_and_files = {}
+    for directory_name in folder_paths:
+        temp_list = []
+        for file in glob.glob(f"{folder_paths[directory_name]}/*.py", recursive=True):
+            if basename(file) != "__init__.py":
+                temp_list.append(basename(file))
+        folders_and_files[directory_name] = temp_list
+    return folders_and_files
+
+
+def extract_doctrings_from_files() -> Dict[str, Dict[str, str]]:
+    """
+    Returns a dictionary containing folder name which in turn
+    is a dictionary containing files and their respective descriptons
+    """
+    total_files = extract_folder_file_structure()
+    folder_file_docstring: Dict[str, Dict[str, str]] = {}
+    for folder, files in sorted(total_files.items()):
+        folder_file_docstring[str(folder)] = {}
+        for file in files:
+            imported_file = importlib.import_module(
+                f'{folder.replace("/",".")}.{file[:-3]}'
+            )
+            extracted_docstring = cast(str, imported_file.__doc__)
+            docstring = extracted_docstring.strip().replace("\n", " ")
+            if len(docstring) >= 104:
+                raise ValueError(f"{file} has docstring longer than 104")
```

Comment:
It's good to avoid hard-coded numbers; it's fairly obvious which variable this comes.